src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
```

Comment:
This method can be improved by directly returning the new 'Ball' object without creating temporary variables for 'color' and 'number'.